main.go

```diff
@@ -67,23 +67,21 @@ func main() {
 }
 
 func setConfig(path string) {
+
 	// Default values
 	viper.SetDefault("host.listen", "")
 	viper.SetDefault("host.port", "4242")
 	viper.SetDefault("host.hook", "hook")
 
-	viper.SetDefault("repo.url", "https://github.com/0rax/fishline.git")
-	viper.SetDefault("repo.path", "/opt/git2etcd/repo")
+	viper.SetDefault("repo.path", "data/")
```

Comment:
Why changing the order here ?

Reply:
No real reason, I might have removed one at some point and added it back after. Both values have been changed in the end.